src/styleq.js

```diff
@@ -9,10 +9,19 @@
 
 'use strict';
 
-import type { IStyleq, Styleq, StyleqOptions } from '../styleq.flow';
+import type {
+  IStyleq,
+  Styleq,
+  StyleqOptions,
+  EitherStyle,
+  InlineStyle,
+  Styles,
+} from '../styleq.flow';
 
-const cache = new WeakMap();
-const compiledKey = '$$css';
+type Cache = WeakMap<EitherStyle, [string, Array<string>, Cache]>;
```

Comment:
I changed `EitherStyle` to `CompiledStyle`, and that's the only type the cache accepts